packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -11,35 +11,32 @@ import 'typography.dart';
 class CircleAvatar extends StatelessComponent {
   CircleAvatar({
     Key key,
-    this.label,
+    this.child,
```

Comment:
I figured, I'm just not sure this is the right way to do it. It makes the code a lot more verbose in the text case. I'd wait until we have an image case before trying to solve that problem.